lib/js/components/Survey-Questions/SurveyQuestionScale/SurveyQuestionScale.stories.mdx

```diff
@@ -80,140 +86,140 @@ import { SURVEY_QUESTION_STATES } from '../SurveyQuestion.consts';
 	}}
 </Story>
 
-<Story
-	name="examples/limitedWidth"
-	parameters={{ controls: { disabled: true } }}
-	args={{
-		title: "Main question write here if it's long it will collapse.",
-		elaborationLabel: 'Jeśli chcesz, uzasadnij lub rozwiń swoją ocenę',
-		elaborationValue: '',
-		explanation:
-			'<h3 class="-headlineL -textRegular" style="text-align: center; margin-bottom: 16px;">Jak ocenić, czy slajdy i diagramy były zrozumiałe?</h3>\n<div>Wyczerpujące materiały dają poczucie pełnego zrozumienia, przy jednoczesnym usystematyzowaniu informacji. Wpływa na to nie tylko ich jakość, ale też ilość.</div>',
-		placeholder: 'Wpisz swoją odpowiedź',
-		scaleOptions: [
-			{
-				value: '1',
-				label: 'Nie zgadzam się',
-				meaning: SURVEY_TOGGLE_MEANINGS.PRIMARY,
-				content: '1',
-			},
-			{
-				value: '2',
-				label: 'Trochę się nie zgadzam',
-				meaning: SURVEY_TOGGLE_MEANINGS.PRIMARY,
-				content: '2',
-			},
-			{
-				value: '3',
-				label: 'Trochę się zgadzam',
-				meaning: SURVEY_TOGGLE_MEANINGS.PRIMARY,
-				content: '3',
-			},
-			{
-				value: '4',
-				label: 'Zgadzam się ',
-				meaning: SURVEY_TOGGLE_MEANINGS.PRIMARY,
-				content: '4',
-			},
-			{
-				value: '5',
-				label: 'Nie wiem',
-				meaning: SURVEY_TOGGLE_MEANINGS.NEUTRAL,
-				standalone: true,
-			},
-		],
-	}}
-	argTypes={{
-		explanation: { control: { type: 'text' } },
-		selectedValue: { control: false },
-		scaleOptions: { control: { type: 'object' } },
-		state: {
-			control: { type: 'select', options: Object.values(SURVEY_QUESTION_STATES) },
-			defaultValue: SURVEY_QUESTION_STATES.DEFAULT,
-		},
-	}}
->
-	{(args) => {
-		return {
-			components: { SurveyQuestionScale },
-			props: Object.keys(args),
-			data() {
-				return { selectedValue: null, elaborationData: this.elaborationValue };
-			},
-			watch: {
-				elaboration() {
-					// elaboration is a prop here, Vue don't like modifying props, so we wrap it into valueData
-					this.elaborationData = this.elaboration;
-				},
-			},
-			template:
-				'<div style="max-width: 600px"><survey-question-scale :title="title" :scale-options="scaleOptions" :elaboration-value="elaborationData" :elaborationLabel="elaborationLabel" :placeholder="placeholder" :selected-value="selectedValue" :state="state" @select-change="selectedValue = $event" @elaboration-change="elaborationData = $event">' +
-				'<div v-if="explanation" slot="explanation" v-html="explanation" />' +
-				'</survey-question-scale></div>',
-		};
-	}}
-</Story>
+        <Story
```

Comment:
@kvas-damian can You check in your IDE is it correct?

Reply:
The problem is in the plugin `MDX`. When I disable it, everything is ok. So I'll align the text only.